org.metaborg.spoofax.meta.core/pom.xml

```diff
@@ -68,6 +68,11 @@
       <groupId>org.metaborg</groupId>
       <artifactId>tableinterfaces</artifactId>
       <version>${metaborg-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.metaborg</groupId>
+      <artifactId>stratego.typed.pack</artifactId>
```

Comment:
As discussed, probably better to call this library something like `stratego.compiler.pack`, as it is not typed.